internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -399,4 +409,106 @@ Resources:
                 Resource: "*"
       ManagedPolicyArns:
         - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+
+  CertificateValidatorAction:
```

Comment:
can we add these to the trigger state machine action `DependsOn` if alias is enabled?

Reply:
i guess i'm not sure how that will work with the `Condition` field here... Would it make sense to not even include these resources in the template if alias is not set?